tutorials/FSS.ipynb

```diff
@@ -0,0 +1,2354 @@
+    "num_cols = 600\n",
+    "num_rows = 400\n",
+    "\n",
+    "# set seed for reproducibility\n",
+    "np.random.seed(42)\n",
+    "\n",
+    "# generate random fields\n",
+    "obs = np.random.normal(loc=0.0, scale=1.0, size = (num_rows, num_cols))\n",
+    "fcst_1 = np.random.normal(loc=0.0, scale=1.0, size = (num_rows, num_cols))\n",
+    "fcst_2 = np.random.normal(loc=1.0, scale=1.0, size = (num_rows, num_cols))\n",
+    "fcst_3 = np.random.normal(loc=0.0, scale=2.0, size = (num_rows, num_cols))\n",
+    "\n",
+    "# print out the different scenarios\n",
+    "_summarize = lambda x, field: print(\n",
+    "    \"{: >20}: shape={}, mean={:.2f}, stddev={:.2f}\".format(\n",
+    "    field, x.shape, np.mean(x), np.std(x)\n",
+    "))\n",
+    "_summarize(obs, \"observations\")\n",
+    "_summarize(fcst_1, \"forecast scenario 1\")\n",
+    "_summarize(fcst_2, \"forecast scenario 2\")\n",
+    "_summarize(fcst_3, \"forecast scenario 3\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "65de79c5-eb7d-49e1-bb74-469d6ef4295e",
+   "metadata": {},
+   "source": [
+    "**2. Define inputs**\n",
+    "\n",
+    "We need to now specify the threshold, window size and compute method. For now, lets choose a single window, and threshold. While the current `fss` method doesn't allow for more than 1 threshold and window definition per call, we'll see how calculate multiple thresholds/windows in a later step."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "ddfb3f97-89f1-4b8d-ba7a-1227d06b8757",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "window_size: (100, 100)\n",
+      "threshold: 0.5\n",
+      "compute_method: FssComputeMethod.NUMPY\n"
+     ]
+    }
+   ],
+   "source": [
+    "window_size = (100, 100)  # height * width or row size * col size\n",
+    "threshold = 0.5  # arbitrarily chosen\n",
+    "\n",
+    "# Note: the compute method is a specialized argument, it doesn't need to be specified but is shown here for illustration purposes.\n",
+    "# If you do not specify a compute method it will default to `NUMPY` which is the only method currently supported and is sufficient\n",
+    "# for most purposes.\n",
+    "from scores.fast.fss.typing import FssComputeMethod\n",
+    "compute_method = FssComputeMethod.NUMPY\n",
+    "\n",
+    "print(\"window_size:\", window_size)\n",
+    "print(\"threshold:\", threshold)\n",
+    "print(\"compute_method:\", compute_method)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4e12430b-f3f9-43b6-bc60-9b1e0b77eb63",
+   "metadata": {},
+   "source": [
+    "**3 Run FSS**\n",
+    "\n",
+    "Since we only have spatial dims it is sufficient to use  `scores.spatial.fss_2d_single_field` for this purpose.\n",
+    "\n",
+    "There are reasons why a user may prefer to use this API:\n",
+    "- The user can choose their own aggregation method\n",
```

Comment:
Consider adding an empty line before the dot points, so the dot points render properly in readthedocs